style.css

```diff
@@ -470,6 +471,56 @@ article h1 {
     width: 684px;
   }
 
+  .frameleftdesk,
+  .framerightdesk {
+    display: flex;
+  }
+
+  .frameleftdesk {
+    position: absolute;
+    left: 0;
+    top: 160px;
+    overflow: hidden;
+    display: inline-block;
+    overflow-x: hidden;
+  }
+
+  .framerightdesk {
+    position: absolute;
+    right: 0;
+    top: -40px;
+    overflow: hidden;
+    display: inline-block;
+    overflow-x: hidden;
+  }
+
+  .contactframe1 {
+    position: absolute;
+    left: 0;
+    bottom: -270px;
+    overflow: hidden;
+    display: inline-block;
+    overflow-x: hidden;
+  }
+
+  .contactframe2 {
+    position: absolute;
+    bottom: -270px;
+    left: 500px;
+    overflow: hidden;
+    display: inline-block;
+    overflow-x: hidden;
+  }
+
+  .contactframe3 {
+    position: absolute;
+    right: 0;
+    bottom: -270px;
+    overflow: hidden;
+    display: inline-block;
+    overflow-x: hidden;
```

Comment:
- [ ] as mentioned in the requirements 
![image](https://user-images.githubusercontent.com/89282221/214825926-34dc59e2-d824-4472-ba2c-b339ee0f1e7e.png)

kindly remove the last name input and change it to only one input that will handle a full name instead 🙏